<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGUI</class>
 <widget class="QMainWindow" name="CGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="evulationScore">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="evLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="performButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Perform</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="imageView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>461</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
